app/routes/location.php

```diff
@@ -24,5 +24,7 @@
 Route::get('location-reporting/{location_id}/{question_category_id}', 'LocationReportController@show2');
 Route::post('api/set-search-radius', 'LocationSearchController@setSearchRadius');
 Route::post('api/add-suggestion', 'SuggestionController@addSuggestion');
+Route::get('suggestion-list/{location_id}','SuggestionController@showSuggestionList');
+Route::get('suggestion-detail/{suggestion_id}','SuggestionController@showSuggestionDetail');
```

Comment:
Please run "composer all" and fix the problems that it finds.

Here is what I get when I run it.
![image](https://user-images.githubusercontent.com/11838191/43256625-4cb831ae-909b-11e8-8d85-83c5cd0a7505.png)